src/chain/member_info.rs

```diff
@@ -7,18 +7,40 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 /// Information about a member of our section.
+
+pub type AgeCounter = u32;
+
+/// The minimum allowed value of the Age Counter
+pub const MIN_AGE: AgeCounter = 16;
+
+const MAX_INFANT_AGE: u8 = 4;
```

Comment:
Well, conceptually this is the maximum age at which we call a node an infant - this could change to not be equal to the starting age, and the `MAX_` would make more sense then. I can change this, though, if you prefer.

Reply:
I am OK with it as is actually @fizyk20